### PR TITLE
Feature/main

### LINKED ISSUE #23: Create bot start script
Delatore can't be started as python module now

### LINKED ISSUE #31: Move all long IO operations to Threads
Currently, handling of commands is done in main thread, this can slow bot down

Please consider move command handling to Threads

### PR DESCRIPTION
**Pull request by outcatcher**
Implement `__main__` for delatore so it can be used as python module (`python -m delatore`)

Resolve #23 
Resolve #31

### CODE REVIEW

File: delatore/__main__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    else:
+        raise RuntimeError("Please provide chat ID and bot token or configuration file to use")
+
+
+def _main():
+    _config()
+    from delatore import bot
+    delatore = bot.Delatore()
+    delatore.start()
+
```
Comment (anton-sidelnikov) [inline delatore/__main__.py]: http server should start here too

Comment (outcatcher) [reply delatore/__main__.py]: IMO servers should be started in `delatore.start()`

Comment (anton-sidelnikov) [reply delatore/__main__.py]: ok

File: delatore/__main__.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        if not os.path.isfile(config_file):
+            raise FileNotFoundError
+        configuration.BOT_CONFIG = configuration.read_config(config_file)
+    else:
+        raise RuntimeError("Please provide chat ID and bot token or configuration file to use")
+
+
+def _main():
+    _config()
+    from delatore.bot import bot
```
Comment (anton-sidelnikov) [inline delatore/__main__.py]: why import here?

File: delatore/bot.py [hunk 3]
```diff
@@ -1,96 +1,108 @@
...
+        self.monitor_sources()
+        self.infinity_polling(interval=self.TG_POLLING_INTERVAL)
+
+
+bot = DelatoreBot(BOT_CONFIG.token)
+
+
+@bot.message_handler(commands=["status"])
+def handle_start_help(message: Message):
+    # msg_text = "/status awx:scenario1 5"
```
Comment (anton-sidelnikov) [inline delatore/bot.py]: delete this

File: delatore/__main__.py (hunk 2 above)
Comment (outcatcher) [reply delatore/__main__.py]: Bot is created on import, it depends on `BOT_CONFIG` populated in our script, so we have to change it and after that import bot

**Pull request merged**